amulet/sentry.py

```diff
@@ -192,16 +192,16 @@ def relation(self, from_rel, to_rel):
 
 # Possibly use to build out instead of having code in setup()?
 class Talisman(object):
-    def __init__(self, services, rel_sentry='relation-sentry', juju_env=None, timeout=300):
```

Comment:
True, we think of it as "internal", but technically speaking it is public b/c it's exported by the sentry module.